### ISSUE TITLE
"Try to search for" on iPhone does not search

### ISSUE THREAD

**Issue by rkra** [nuxsmin/sysPass#1172]
**sysPass Version**
DEMO RC8

**Describe the bug**
"Try to search for" on iPhone does not start the search

**To Reproduce**
1. Fill is "try to search for"
2. There is no "Search" Button. The search does not start pressing the iPhone "Done" oder "Return" button.

**Comment by anthillinc** [comment]
Hello,

I have the same issue. I have tested iPhone and iPad and Safari - all are the same as you describe.

I also tested Chrome and MS Edge on iPhone, same issue... it's as if the interface does not recognize the "enter" button after entering text on the text search field. Which is strange because the enter button does work on the main login screen.

Our main reason for upgrading to V3 (most recent version) is to get a responsive interface, now it seems we can't use it on devices that need responsive.

Thanks, Daniel.
